scripts/packaging/polkadot.service

```diff
@@ -12,6 +12,27 @@ Restart=always
 RestartSec=120
 MemoryHigh=5400M
 MemoryMax=5500M
+CapabilityBoundingSet=
+LockPersonality=true
+NoNewPrivileges=true
+PrivateDevices=true
+PrivateMounts=true
+PrivateTmp=true
+PrivateUsers=true
+ProtectClock=true
+ProtectControlGroups=true
+ProtectHostname=true
+ProtectKernelModules=true
+ProtectKernelTunables=true
+ProtectSystem=strict
+RemoveIPC=true
+RestrictAddressFamilies=AF_INET AF_INET6 AF_NETLINK AF_PACKET
```

Comment:
We should not need `AF_PACKET`.  In this case, it is harmless, as it requires `CAP_NET_RAW` to use, but it is still good practice to avoid it.